integrations/gitlab/.port/spec.yaml

```diff
@@ -29,3 +29,7 @@ configurations:
     type: boolean
     description: If set to true, will use system hook instead of project hooks.
     default: false
+  - name: groupsPathsForHooks
+    required: false
+    type: array
+    description: A list of full paths of specified groups to open webhooks for on start. 
```

Comment:
currently users can pass multiple tokens for each group path, I think we should align the behavior of the webhook creation to be the same

{"token1": ["group/path1", "group/path2"]} etc..

Reply:
i think we talked about it on Thursday
if you would like, lets talk about it again tomorrow @Tankilevitch

Reply:
done